### PR TITLE
[WIP] Decompose commands into independent chapters

### PR DESCRIPTION
**Pull request by jessicard**
## Command Line Basics Part One chapters
- [x] Remove Command Line Basics Part One (https://github.com/zincmade/just-enough-unix-command-line/pull/56)
- [x] Introduction (https://github.com/zincmade/just-enough-unix-command-line/pull/48)
- [ ] The Terminal (https://github.com/zincmade/just-enough-unix-command-line/pull/51)
- [ ] The File System (https://github.com/zincmade/just-enough-unix-command-line/pull/54)
- [ ] `pwd` (https://github.com/zincmade/just-enough-unix-command-line/pull/52)
- [ ] `ls` (https://github.com/zincmade/just-enough-unix-command-line/pull/53)
- [ ] `cd` (https://github.com/zincmade/just-enough-unix-command-line/pull/55)

## Command Line Basics Part Two chapters
- [ ] Remove Command Line Basics Part Two
- [ ] `mkdir`
- [ ] `touch`
- [ ] `vi`
- [ ] `cat`
- [ ] `cp`
- [ ] `mv`
- [ ] `rm`, `rmdir`, `rm -rf`

### CODE REVIEW

**Comment by zspencer** [comment]
I'm thinking that we may not want to pull out each command; seeing as they're so short? Maybe something like:

```
Get your bearings - (the terminal, the file system, pwd, ls)
Go exploring - (cd, cat, `ls -F`, man) 
Make your mark - (mkdir, touch, nano, cp, mv)
Cleaning up - (rm, rmdir, rm -rf)
```

Alternatively:
Get your bearings - (the terminal, the file system, pwd, ls)
Go exploring - (cd, cat, `ls -F`, man) 
Working with files - `(touch, nano, cp, mv, rm)
Working with directories - (mkdir, rmdir, rm -rf, cp -r)
```


```

File: en-US/src/introduction.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Introduction
+
+One thing that separates advanced practitioners from beginners is the comfort and ease with which they use the UNIX command line. The command line, while intimidating, can make many tasks easier and faster. Learning the command line allows you to speak directly with the computer instead of having to wade through with a translator.
+
+While it may seem like we're starting off slowly, we have to be able to walk before we can run. The first few chapters make sure you're comfortable with the fundamentals. The middle includes more intermediate theory and commands. The final chapters will demonstrate how to connect the principles behind the UNIX command line and the built-in commands on your computer.
+
+For those of you using a Mac or Linux computer, you have the UNIX command line already! Congratulations! For those of you on Windows, you'll need to install [Linux within a Virtual Machine](http://www.wikihow.com/Install-Ubuntu-on-VirtualBox).
+
+This book will assume you are using a Mac. Although you can use the command line on any machine, some specific directions may vary depending on the operating system you're using.
```
Comment (zspencer) [inline en-US/src/introduction.md]: I'd prefer if we didn't assume people were on a mac, as that makes may alienate people without access to linux machines :(. Can we make it a goal to provide both mac and linux instructions when possible?

**Comment by jessicard** [comment]
FYI - Hoping to do more work on this this weekend!

**Comment by zspencer** [comment]
:+1: Looking forward to leaving feedback :D

**Pull request closed without merging**